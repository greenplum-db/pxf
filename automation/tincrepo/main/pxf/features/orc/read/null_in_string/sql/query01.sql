-- @description query01 for reading strings contain NUL-byte from ORC files

SELECT * FROM pxf_orc_null_in_string ORDER BY id;
