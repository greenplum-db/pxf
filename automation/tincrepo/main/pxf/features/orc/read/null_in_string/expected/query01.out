-- start_ignore
-- end_ignore
-- @description query01 for reading strings contain NUL-byte from ORC files
SELECT * FROM pxf_orc_null_in_string ORDER BY id;
 id |               context               |     value
----+-------------------------------------+---------------
  1 | simple string                       | hello
  2 | simple string with space            | hello world
  3 | simple string with double quote     | hello "world"
  4 | NUL-byte in middle of string        | hello
  5 | NUL-byte at the beginning of string |
  6 | NUL-byte at the end of string       | hello world
(6 rows)